### PR TITLE
using cloud_audience_urn from user realm response as value for applie…

### PR DESCRIPTION
**Pull request by SomkaPe**
WS-Trust does not work against sovereign clouds e.g. - BlackForest https://github.com/AzureAD/azure-activedirectory-library-for-java/issues/113

### CODE REVIEW

File: src/main/java/com/microsoft/aad/adal4j/WSTrustRequest.java [hunk 1]
```diff
@@ -157,7 +156,7 @@ private static StringBuilder buildMessage(String address, String username,
                                 + "</s:Envelope>", schemaLocation, soapAction,
                                 guid, address,
                                 securityHeaderBuilder.toString(),
-                                rstTrustNamespace, DEFAULT_APPLIES_TO, keyType,
+                                rstTrustNamespace, cloudAudienceUrn, keyType,
```
Comment (kpanwar) [inline src/main/java/com/microsoft/aad/adal4j/WSTrustRequest.java]: what if the cloudAudienceUrn is null? we should use default value in case of null.

File: src/main/java/com/microsoft/aad/adal4j/WSTrustRequest.java [hunk 2]
```diff
@@ -40,10 +40,9 @@
             .getLogger(WSTrustRequest.class);
 
     private final static int MAX_EXPECTED_MESSAGE_SIZE = 1024;
-    private final static String DEFAULT_APPLIES_TO = "urn:federation:MicrosoftOnline";
-
+    
     static WSTrustResponse execute(String url, String username,
```
Comment (kpanwar) [inline src/main/java/com/microsoft/aad/adal4j/WSTrustRequest.java]: add a test case for null and non null cloud audience urn

Comment (SomkaPe) [inline src/main/java/com/microsoft/aad/adal4j/WSTrustRequest.java]: added

File: src/main/java/com/microsoft/aad/adal4j/WSTrustRequest.java (hunk 1 above)
Comment (SomkaPe) [inline src/main/java/com/microsoft/aad/adal4j/WSTrustRequest.java]: changed
